packages/compartment-mapper/src/types.js

```diff
@@ -60,6 +61,7 @@ export {};
  * @typedef {Object} ModuleDescriptor
  * @property {string=} [compartment]
  * @property {string} [module]
+ * @property {string} [policyId]
```

Comment:
It seems this property isn't used anymore.  If so, please delete.

```suggestion
```

Reply:
thanks for spotting this! leftover from uglier implementation